backend/bots/discord/discordBot.py

```
@@ -0,0 +1,234 @@
+import discord
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)